packages/react/src/components/cookieConsent/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './modal/Modal';
```

Comment:
Remember to add here all functions and components that we wish to expose to public.